src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -224,6 +224,11 @@ public override void VisitLocalFunctionStatement(LocalFunctionStatementSyntax no
                 var oldMethod = _containingMemberOrLambda;
                 _containingMemberOrLambda = match;
 
+                foreach (var parameter in node.ParameterList.Parameters)
+                {
+                    Visit(parameter.Default);
```

Comment:
It does look like we have a potential inconsistency here with respect to this flag.